R/LinReg.R

```diff
@@ -1,6 +1,6 @@
 LinReg<- function(model, particles=1000, plot=FALSE) {
 
-    data(radiata)
+    data(radiata, envir = environment())
```

Comment:
I've tried. This passes all of the sanity checks I tried with R 3.4.1 and Travis seems happy with it.